### PR TITLE
CriteriaVideo class added

### PR DESCRIPTION
**Pull request by MadayAlcala**
Hi guys, I added the 'CriteriaVideo' class, please review it.

### CODE REVIEW

File: src/main/java/com/fundation/webservice/model/CriteriaVideo.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
+package com.fundation.webservice.model;
+
+public class CriteriaVideo extends CriteriaConvert {
+
```
Comment (alszla) [inline src/main/java/com/fundation/webservice/model/CriteriaVideo.java]: It is advisable to remove this line.

File: src/main/java/com/fundation/webservice/model/CriteriaVideo.java [hunk 2]
```diff
@@ -0,0 +1,90 @@
+package com.fundation.webservice.model;
```
Comment (alszla) [inline src/main/java/com/fundation/webservice/model/CriteriaVideo.java:10]: Please add the provided copyright header.

File: src/main/java/com/fundation/webservice/model/CriteriaVideo.java [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
```
Comment (alszla) [inline src/main/java/com/fundation/webservice/model/CriteriaVideo.java:11]: Please add a custom description of this class.

File: src/main/java/com/fundation/webservice/model/CriteriaVideo.java (hunk 1 above)
Comment (MadayAlcala) [reply src/main/java/com/fundation/webservice/model/CriteriaVideo.java]: solved

File: src/main/java/com/fundation/webservice/model/CriteriaVideo.java (hunk 2 above)
Comment (MadayAlcala) [reply src/main/java/com/fundation/webservice/model/CriteriaVideo.java:10]: solved

File: src/main/java/com/fundation/webservice/model/CriteriaVideo.java (hunk 3 above)
Comment (MadayAlcala) [reply src/main/java/com/fundation/webservice/model/CriteriaVideo.java:11]: solved

**Pull request merged**